client/src/unifyfs.c

```diff
@@ -858,15 +858,25 @@ int unifyfs_gfid_stat(int gfid, struct stat* buf)
     size_t filesize;
     ret = invoke_client_filesize_rpc(gfid, &filesize);
     if (ret != UNIFYFS_SUCCESS) {
-        return ret;
```

Comment:
There could be a case where the file has been laminated, but the filesize_rpc failed with as error.  For that case, let's still have this return an error.  Can we use the laminated field on the metadata to distinguish instead?

Reply:
In my next push, I'll cache the size on lamination so this won't be an issue.